spec/delve_spec.vim

```diff
@@ -61,11 +61,17 @@ describe "Delve"
     after
       unlet g:test#go#delve#options
     end
-    it "appends options to the end"
+    it "appends options to the end in a suite test"
       view normal_test.go
       TestSuite
 
-      Expect g:test#last_command == 'dlv test ./... -test.v'
+      Expect g:test#last_command == 'dlv test ./... -- -test.v'
+    end
+    it "appends options to the end in a nearest test"
```

Comment:
This test exists to ensure the `--` is not added twice, since it can be added between the package name and the `-test.run` which are power of the bills position, as well as before the custom options `-test.v`.